dataplicity/m2mmanager.py

```diff
@@ -118,27 +127,35 @@ def run(self):
 
         while 1:
             # Get the identity, and tell the server about it
+            log.debug('1')
```

Comment:
Forgive these debug lines for now. I'll keep them in until we're more confident in the logic.